tests/tracing/test_fluent.py

```diff
@@ -119,6 +137,67 @@ def test_trace(with_active_run):
     }
 
 
+@pytest.mark.parametrize("with_active_run", [True, False])
```

Comment:
nit: I'd just add a new parameter in `test_trace` to ensure the assertions are exactly same between sync and async tests.

Reply:
Good point. That was my original thought but then I wanted to make the test itself async. Your comment made me realise that I can just wrap the call in an `asyncio.run`, so that's what I've done now.